app/models/swb/person.rb

```diff
@@ -30,6 +30,8 @@ module Swb::Person
     }
 
     alias_method :member_id, :id
+
+    before_validation :reset_ts_gender, unless: -> { gender.blank? }
```

Comment:
```suggestion
    before_validation :sync_ts_gender, unless: -> { gender.blank? }
```

Reply:
`sync` haben wir eher fürs Übermitteln der infos an umsysteme, `reset` finde ich in diesem Fall besser, da der abhängige `ts_gender` wert zurückgesetzt wird, wenn er nicht blank ist (was TS nicht kennt)